### PR TITLE
8-7. Album (Alexa)

### PR DESCRIPTION
**Pull request by Bookwormlex**

### CODE REVIEW

**Comment by therawv** [comment]
Your file isn't in the chapter_08 directory.

File: 8-7.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        }
+    if song_count:
+        album["Song Count"] = song_count
+    return album
+
+print(make_album("Nanna", "How to Start a Garden"))
+print(make_album("Maddie Zahm", "Fat Funny Friend"))
+print(make_album("Yellowcard", "Ocean Avenue"))
+
+print(make_album("Flamy Grant", "Bible Belt Baby", 10))
```
Comment (therawv) [inline 8-7.py:16]: This is functional but could be formatted to be more legible. Up to you.

**Review by therawv — CHANGES_REQUESTED** [review]
Move the file to the correct directory.

File: 8-7.py (hunk 1 above)
Comment (Bookwormlex) [reply 8-7.py:16]: Gonna pass, but thanks for the note.

**Comment by Bookwormlex** [comment]
Fixed!

**Review by therawv — APPROVED** [review]

**Pull request merged**